### ISSUE TITLE
[Bug] NPCs with crossbows unable to Reload or use their crossbow attack in general

### ISSUE THREAD

**Issue by Ghostdesu** [JDCalvert/FVTT-PF2e-Ranged-Combat#8]
**Environment**
FoundryVTT 9.245
pf2e 3.3.0.10241
libWrapper 1.12.2.0
PF2e Ranged Combat 2.0.0

**Describe the issue**
Prevent Firing Weapon if not Loaded applies to NPC sheets despite them being unable to reload. 
Dragging the Loaded effect directly on the token does not seem to work either.

**To Reproduce**
1. Import any NPC creature with a crossbow. (Bounty Hunter from NPC Gallery was used for testing).
2. Import the Reload macro.
3. Try to use its crossbow attack.
4. Try to use Reload macro.
5. Try to use the attack again.

**Expected behavior**
Ideally the Reload macro would work with NPC sheets, alternatively Prevent Firing Weapon if not Loaded should ignore them.

**Screenshots**
![cantreload](https://user-images.githubusercontent.com/31375910/151696777-a3630b0c-7faf-4c17-a9d6-a92936d6dc43.gif)
![image](https://user-images.githubusercontent.com/31375910/151696755-fbdd73d7-355c-4154-8ff2-31a1f46f95b5.png)

**Comment by Ghostdesu** [comment]
```
Uncaught (in promise) TypeError: (intermediate value).requiredLoading is not a function
[Detected 1 package: pf2e-ranged-combat]
    weapons http://localhost:32000/modules/pf2e-ranged-combat/scripts/actions/reload.js:143
    getReloadableWeapons http://localhost:32000/modules/pf2e-ranged-combat/scripts/actions/reload.js:143
    reload http://localhost:32000/modules/pf2e-ranged-combat/scripts/actions/reload.js:24
    anonymous http://localhost:32000/scripts/foundry.js line 16907 > Function:4
    anonymous http://localhost:32000/scripts/foundry.js line 16907 > Function:5
    _executeScript http://localhost:32000/scripts/foundry.js:16909
    execute http://localhost:32000/scripts/foundry.js:16866
    _onClickMacro http://localhost:32000/scripts/foundry.js:52252
    jQuery 2
        dispatch
        handle
reload.js:143:48
    reload http://localhost:32000/modules/pf2e-ranged-combat/scripts/actions/reload.js:86
    anonymous http://localhost:32000/scripts/foundry.js line 16907 > Function:4
    anonymous http://localhost:32000/scripts/foundry.js line 16907 > Function:5
    _executeScript http://localhost:32000/scripts/foundry.js:16909
    execute http://localhost:32000/scripts/foundry.js:16866
    _onClickMacro http://localhost:32000/scripts/foundry.js:52252
    _onClickMacro self-hosted:1225
    jQuery 2
        dispatch
        handle
```

**Comment by JDCalvert** [comment]
Thank you for reporting this issue (and for the amazing detail!). Should be fixed as of version 2.0.1.
